telegram/files/file.py

```diff
@@ -89,15 +89,15 @@ def __init__(
         self.file_unique_id = str(file_unique_id)
         # Optionals
         self.file_size = file_size
-        self.file_path: Optional[str] = file_path  # Can be either path or url, so must remain str.
+        self.file_path = file_path  # Can be either path or url, so must remain str.
```

Comment:
please remove the comment - it doesn't add any additional information

Reply:
done